grafana/grafana
# Public Dashboards: Query Caching

**What this does**
This is an enterprise-only feature. Queries for panels that come from a public dashboard will be cached the same way queries from regular dashboard panels do. [Enterprise PR](https://github.com/grafana/grafana-enterprise/pull/3518)

**How to test**
- Enable public dashboards and query caching in your ini
`publicDashboards = true`
`caching.queries = true`
- Enable caching for the datasource(s) that your public dashboard uses
- Check your network tab for the `query` request when running a public dashboard. After you refresh the page, the `X-Cache` header should show as a `HIT`
